oak-core/src/test/java/org/apache/jackrabbit/oak/namepath/impl/NamePathMapperImplTest.java

```diff
@@ -135,6 +135,26 @@ public void testOakToJcr() {
         }
     }
 
+    @Test
+    public void testOakToExpandedJcr() {
+        assertEquals("/{http://www.example.com/foo}bar", npMapper.getExpandedJcrPath("/oak-foo:bar"));
+        assertEquals("/{http://www.example.com/foo}bar/{http://www.example.com/quu}qux", npMapper.getExpandedJcrPath("/oak-foo:bar/oak-quu:qux"));
+        assertEquals("{http://www.example.com/foo}bar", npMapper.getExpandedJcrPath("oak-foo:bar"));
+        assertEquals(".", npMapper.getExpandedJcrPath(""));
```

Comment:
"" -> "."?

I assume this is intended, but maybe add a comment why?

Reply:
This has been introduced with OAK-6 by you: https://github.com/apache/jackrabbit-oak/commit/79f71cfa2f567ebc778e8eb962a9a44619958369

TBH I don't understand it either.

Reply:
This is the self-segment defined in https://s.apache.org/jcr-2.0-spec/3_Repository_Model.html#3.4.2%20Path%20Resolution. This is also valid in expanded name form for paths.